src/sessions/session-manager.ts

```diff
@@ -286,6 +286,11 @@ export class SessionManager {
           { existingSessionId: existingSession.sessionId, proposeSessionId: sessionInfo.sessionId, conversationId: sessionInfo.conversationId});
         existingSession.sessionId = sessionInfo.sessionId;
         existingSession.id = sessionInfo.id;
+
+        if (existingSession.accepted) {
```

Comment:
this is the gist of the fix. If we have a pendingSession that just had its id updated, we check to see if it was accepted (answered/proceeded). If it was, we send the proceed again so it goes over xmpp rather than rest api.